### DISCUSSION TITLE
Install

### DISCUSSION THREAD

**Discussion by ghost** [blueedgetechno/win11React#104] [General]
How install this project on my localhost ?

i need testing on my computer to adding any ideas

i need only Explain install

because i get this

![9v](https://user-images.githubusercontent.com/73659109/131958142-4e2ba7d5-4cd3-4865-8f4b-0856cb2ab757.PNG)

Thanks

**Comment by blueedgetechno** [comment]
You need to use npm.

step 1:
```
git clone https://github.com/blueedgetechno/windows11
```

step 2:
```
cd windows11
```

step 3:
```
npm install
```

step 4:
```
npm start
```

**Reply by ghost** [reply]
are this normal ? 

![9m](https://user-images.githubusercontent.com/73659109/131986305-f1991cdb-48db-42f6-a859-cdfcd9ea412e.PNG)

**Reply by ghost** [reply]
after finish, i get error what can i do to solving this ?

![4s](https://user-images.githubusercontent.com/73659109/131987054-bf0ba559-8736-4e89-acbe-ca5466457bee.PNG)

**Reply by ghost** [reply]
are u see ? 

![98](https://user-images.githubusercontent.com/73659109/131990651-9db8133c-efe4-4897-af92-f484192901df.PNG)


what can do to sovling it ?

**Reply by blueedgetechno** [reply]
check your npm version

**Reply by Diddi81Ma** [reply]
> check your npm version

**Reply by andrewstech** [reply]
@mazenleno10x If you still can't get it to work. Then we have a windows version that is pre compiled. https://github.com/blueedgetechno/windows11/releases
